package.json

```diff
@@ -21,29 +21,29 @@
     "test": "jest",
     "test:cover": "jest --coverage",
     "test:cover:CI": "cat coverage/lcov.info | coveralls",
-    "lint": "eslint '**/*.js' --fix",
+    "lint": "eslint **/*.js --fix",
```

Comment:
You might want to test this one, CI failed during the lint task and I seem to remember the quotes were required for some reason?

Reply:
@DamianMullins Yup, I didn't realise I'd committed this line! Testing now with escaped `"`s